corehq/apps/export/dbaccessors.py

```diff
@@ -0,0 +1,32 @@
+
+
+def get_latest_case_export_schema_id(domain, case_type):
+    from .models import ExportDataSchema
+
+    key = [domain, 'CaseExportDataSchema', case_type]
+    result = ExportDataSchema.get_db().view(
+        'schemas_by_xmlns_or_case_type/view',
+        startkey=key + [{}],
+        endkey=key,
+        include_docs=False,
+        limit=1,
+        reduce=False,
+        descending=True,
```

Comment:
we shouldn't have to do that. on editing/view we don't even have to run the `generate_from_builds` call, but if we decide we do, we'll only create a new object if there is a new build. only `_merge_schemas` returns a new object. otherwise the old one should be used: https://github.com/dimagi/commcare-hq/blob/save-export-state/corehq/apps/export/models/new.py#L369